CHANGELOG.md

```diff
@@ -10,9 +10,30 @@
 
 ### Features
 
-- Adds the `FeedbackButton` component that shows the Feedback Widget ([#4378](https://github.com/getsentry/sentry-react-native/pull/4378))
-- Adds the `ScreenshotButton` component that takes a screenshot ([#4714](https://github.com/getsentry/sentry-react-native/issues/4714))
-- Add Feedback Widget theming ([#4677](https://github.com/getsentry/sentry-react-native/pull/4677))
+- User Feedback Widget Updates
+  - `FeedbackButton` for easy access to the widget ([#4378](https://github.com/getsentry/sentry-react-native/pull/4378))
+  - `ScreenshotButton` for capturing the application visuals ([#4714](https://github.com/getsentry/sentry-react-native/issues/4714))
+  - Theming support to better align with the application styles ([#4677](https://github.com/getsentry/sentry-react-native/pull/4677))
+
+  ```js
+  Sentry.init({
+    integrations: [
+      Sentry.feedbackIntegration({
+        enableTakeScreenshot: true, // Enables `ScreenshotButton`
+        themeDark: {
+          // Add dark theme styles here
+        },
+        themeLight: {
+          // Add light theme styles here
+        },
+      }),
+    ],
+  });
+
+  Sentry.showFeedbackButton();
+  Sentry.hideFeedbackButton();
+  ```
+
```

Comment:
nit: We could also add a link to the updated documentation https://github.com/getsentry/sentry-docs/pull/13512

```suggestion

  To learn more visit [the documentation](https://docs.sentry.io/platforms/react-native/user-feedback).

```